### PR TITLE
feat: [CDS-74727]: add icons for trigger explorer page

### PR DESCRIPTION
**Pull request by sonaligoyal-harness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/trigger-explorer2.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....075.466.226.626.142.15.419.313.97.313h2.815v-.283a1.2 1.2 0 0 1 1.2-1.2h3.448a1.2 1.2 0 0 1 1.2 1.2v1.382a1.2 1.2 0 0 1-1.2 1.2h-3.448a1.2 1.2 0 0 1-1.196-1.1H6.878c-.803 0-1.424-.247-1.841-.687-.409-.431-.555-.977-.555-1.451V9.986H3.61a1.2 1.2 0 0 1-1.2-1.2V7.403Zm4.648 0H3.61v1.383h3.448V7.403Zm7.26 0h-3.425v1.3h3.426v-1.3Zm-3.425 4.513h3.448v1.382h-3.448v-1.382Z"/><path d="M17.29 2.412a1.8 1.8 0 0 1 1.8 1.8v9.056a.6.6 0 0 1-1.2 0V4.212a.6.6 0 0 0-.6-.6H5.405a.6.6 0 0 0-.6.6v.163a.6.6 0 1 1-1.2 0v-.163a1.8 1.8 0 0 1 1.8-1.8H17.29ZM14.68 18.46a.6.6 0 0 1-.6.6H5.404a1.8 1.8 0 0 1-1.8-1.8v-2.798a.6.6 0 1 1 1.2 0v2.798a.6.6 0 0 0 .6.6h8.675a.6.6 0 0 1 .6.6Z"/><path d="M18.49 12.669a.6.6 0 0 1 .6.6v1.591a4.2 4.2 0 0 1-4.2 4.2h-3.542a.6.6 0 1 1 0-1.2h3.542a3 3 0 0 0 3-3V13.27a.6.6 0 0 1 .6-.6Z"/><path d="M19.871 6.124a1.8 1.8 0 0 1 1.8 1.8v8.326a.6.6 0 1 1-1.2 0V7.924a.6.6 0 0 0-.6-.6H18.49a.6.6 0 1 1 0-1.2h1.381Zm-2.452 14.949a.6.6 0 0 1-.6.6h-8.32a1.8 1.8 0 0 1-1.8-1.8v-1.278a.6.6 0 1 1 1.2 0v1.277a.6.6 0 0 0 .6.6h8.32a.6.6 0 0 1 .6.6Z"/><path d="M21.071 15.65a.6.6 0 0 1 .6.6v1.223a4.2 4.2 0 0 1-4.2 4.2h-3.286a.6.6 0 0 1 0-1.2h3.286a3 3 0 0 0 3-3V16.25a.6.6 0 0 1 .6-.6Z"/></g></svg>
```
Comment (PratyushGar) [inline packages/icons/src/trigger-explorer2.svg:1]: utilise viewbox and remove height and width in the svg

Comment (PratyushGar) [inline packages/icons/src/trigger-explorer2.svg:1]: Can we rename it from explorer1,explorer2 to be able to search similar icons easily

File: packages/icons/src/trigger-explorer-arrow.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="60" height="12" fill="none"><path fill="#AE82FC" d="M59.53 6.53a.75.75 0 0 0 0-1.06L54.757.697a.75.75 0 0 0-1.06 1.06L57.939 6l-4.242 4.243a.75.75 0 0 0 1.06 1.06L59.53 6.53ZM0 6.75h59v-1.5H0v1.5Z"/></svg>
```
Comment (ishantmehtaharness) [inline packages/icons/src/trigger-explorer-arrow.svg:1]: fill should be avoided and not hard coded here

File: packages/icons/src/trigger-explorer1.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none"><path fill="#6938C0" fill-rule="evenodd" d="M4.339 4.461a.707.707 0 0 0-.697.718v12.916c0 .396.312.717.697.717h15.319a.707.707 0 0 0 .696-.717V8.766c0-.396.312-.717.697-.717.384 0 .696.321.696.717v9.329c0 1.188-.935 2.152-2.089 2.152H4.338c-1.153 0-2.088-.964-2.088-2.152V5.179c0-1.19.935-2.153 2.089-2.153h12.185c.385 0 .697.321.697.718a.707.707 0 0 1-.697.717H4.34ZM7.25 8.355a.683.683 0 0 1 .97-.176l3.278 2.364a.913.913 0 0 1 0 1.47L8.22 14.377a.683.683 0 0 1-.97-.176.731.731 0 0 1 .171-1l2.667-1.923L7.42 9.354a.732.732 0 0 1-.171-.999Zm6.14 4.717a.707.707 0 0 0-.696.717c0 .397.312.718.697.718h3.481a.707.707 0 0 0 .697-.718.707.707 0 0 0-.697-.717h-3.481Z" clip-rule="evenodd"/></svg>
```
Comment (ishantmehtaharness) [inline packages/icons/src/trigger-explorer1.svg:1]: fill colors should be avoided here
and should be passed from parent

**Review by ishantmehtaharness — COMMENTED** [review]
Added some comments regarding icons

File: packages/icons/src/trigger-explorer2.svg (hunk 1 above)
Comment (sonaligoyal-harness) [reply packages/icons/src/trigger-explorer2.svg:1]: Sure

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Pull request merged**